.github/workflows/tmate_debug/action.yml

```diff
@@ -0,0 +1,20 @@
+name: "Tmate debugging tests"
+description: "Setup tmate session if the test fails"
+inputs:
+  use-tmate:
+    description: "boolean for enabling TMATE"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: consider debugging
+      shell: bash --noprofile --norc -eo pipefail -x {0}
+      run: |
+        # Enable tmate only in the Rook fork, where the USE_TMATE secret is set in the repo, or if the action is re-run
+        if [ "$GITHUB_REPOSITORY_OWNER" = "rook" ] || [ -n "${{ inputs.use-tmate }}" ] || [ "$GITHUB_RUN_ATTEMPT" -gt 1 ]; then
```

Comment:
Is the tmate session behaved the same as it was before for the failed tests

Reply:
yes: https://github.com/rook/rook/actions/runs/4231680932/jobs/7350498692

Reply:
Yes, it also ran for at roughly 60 minutes, see https://github.com/rook/rook/actions/runs/4231724283/jobs/7392465051